node/src/accountant/mod.rs

```diff
@@ -1742,129 +1688,105 @@ pub mod tests {
             },
             retrieve_transactions_message
         );
-
-        accountant_awaiter.await_message_count(1);
-        let received_payments_recording = accountant_recording_arc.lock().unwrap();
-        let received_payments_message =
-            received_payments_recording.get_record::<ReceivedPayments>(0);
-        assert_eq!(
-            &ReceivedPayments {
-                payments: expected_transactions
-            },
-            received_payments_message
-        );
     }
 
     #[test]
     fn accountant_logs_if_no_transactions_were_detected() {
         init_test_logging();
-        let earning_wallet = make_wallet("earner3000");
-        let blockchain_bridge = Recorder::new().retrieve_transactions_response(Ok(vec![]));
-        let blockchain_bridge_awaiter = blockchain_bridge.get_awaiter();
-        let blockchain_bridge_recording = blockchain_bridge.get_recording();
-        let (accountant_mock, _, accountant_recording_arc) = make_recorder();
+        let (blockchain_bridge, _, blockchain_bridge_recording_arc) = make_recorder();
+        let blockchain_bridge = blockchain_bridge.retrieve_transactions_response(Ok(vec![]));
```

Comment:
Could be also redundant now - basically the whole mechanism behind this could disappear - it is placed at Recorder in recorder.rs, I believe.